tests/www/test_views.py

```diff
@@ -301,6 +302,27 @@ def test_import_variables_success(self):
         self.check_content_in_response('4 variable(s) successfully updated.', resp)
 
 
+class PluginOperator(BaseOperator):
+    pass
+
+
+class TestPluginView(TestBase):
+    def setUp(self):
+        super().setUp()
+
+    def tearDown(self):
+        super().tearDown()
+
+    def test_should_list_plugins_on_page_with_details(self):
+        resp = self.client.get('/plugin')
+        self.check_content_in_response("test_plugin", resp)
```

Comment:
Yay, tests! 😀

Does this cover both plugin sources?

Reply:
I've added tests for entrypoint plugins as well now.